api_tests/schema_responses/views/test_schema_response_detail.py

```diff
@@ -0,0 +1,126 @@
+import pytest
+
+from osf_tests.factories import (
+    RegistrationFactory,
+    AuthUserFactory
+)
+
+from osf.models import SchemaResponse
+from osf_tests.utils import DEFAULT_TEST_SCHEMA
+
+
+@pytest.mark.django_db
+class TestSchemaResponseDetail:
+
+    @pytest.fixture()
+    def user(self):
+        return AuthUserFactory()
+
+    @pytest.fixture()
+    def registration(self, user):
+        return RegistrationFactory(creator=user)
+
+    @pytest.fixture()
+    def payload(self):
+        return {
+            'data': {
+                'type': 'schema-responses',
+                'attributes': {
+                    'revision_response': {
+                        'q1': {'value': 'update value'},
```

Comment:
what's up with this format? Should just be {key: value}, as per the Draft Registration and the API contract [here](https://docs.google.com/document/d/1Z4gDj8PBxlv0JfT_AvGupjlxAamcU7guSUj33r1KaiQ/edit#heading=h.nbocx6st0lrp). Tests will pass for now, I guess, since we're storing arbitrary json, but this will break whenever we add validation to the block.